src/schedule-and-details/details-section/index.jsx

```diff
@@ -57,7 +57,7 @@ DetailsSection.propTypes = {
     PropTypes.arrayOf(PropTypes.string.isRequired).isRequired,
   ).isRequired,
   onChange: PropTypes.func.isRequired,
-  isEditable: PropTypes.bool,
+  isEditable: PropTypes.bool.isRequired,
```

Comment:
You can remove `isEditable` from the `defaultProps` section if it is now required.